com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/UriUtil.java

```diff
@@ -26,6 +30,65 @@
     public static final String HTTP_PROTOCOL = "http";
     public static final String HTTPS_PROTOCOL = "https";
 
+    public static Map<String, Object> splitUri(String uri) {
```

Comment:
This needs to be extracted in a separate class (like Android's [Uri](https://developer.android.com/reference/android/net/Uri)class).